src/App.tsx

```diff
@@ -1,120 +1,75 @@
-    const count = e.target.valueAsNumber;
-
-    if (!isValidPassengerCount(count)) {
-      return;
-    }
-
-    setPassengerCount(count);
-    setMessage(
-      `${PASSENGER_CHANGE_MESSAGE} ${PASSENGER_COUNT_INPUT_ALERT_MESSAGE(
-        count
-      )}`
-    );
-  };
-
-  const increasePassengerCount = () => {
-    const currentPassengerCount = passengerCount;
-
-    if (!isValidPassengerCount(passengerCount + 1)) {
-      return;
-    }
-
-    setPassengerCount((prev) => prev + 1);
-    setMessage(
-      `${PASSENGER_INCREASE_MESSAGE}
-        ${PASSENGER_COUNT_INPUT_ALERT_MESSAGE(currentPassengerCount + 1)}`
-    );
-  };
-
-  const decreasePassengerCount = () => {
-    const currentPassengerCount = passengerCount;
-
-    if (!isValidPassengerCount(passengerCount - 1)) {
-      return;
-    }
-
-    setPassengerCount((prev) => prev - 1);
-    setMessage(
-      `${PASSENGER_DECREASE_MESSAGE}
-        ${PASSENGER_COUNT_INPUT_ALERT_MESSAGE(currentPassengerCount - 1)}`
-    );
-  };
-
   return (
-    <main>
-      <h1>웹 접근성 좋은 항공사</h1>
-      <h2>승객 선택</h2>
-      <section className="passenger-count-control-container">
-        <div className="flex g7 align-center">
-          <label htmlFor="passenger-count-input">성인</label>
-          <QuestionMarkTooltip description="승객은 최소 1명 최대 3명입니다." />
-        </div>
-        <div className="passenger-count-controller">
-          <button
-            className="passenger-count-control-button"
-            type="button"
-            aria-disabled={passengerCount === MIN_PASSENGER_COUNT}
-            aria-label="성인 승객 한명 줄이기"
-            onClick={decreasePassengerCount}
-          >
-            -
-          </button>
-          <input
-            className="passenger-count-control-input"
-            id="passenger-count-input"
-            type="number"
-            min={MIN_PASSENGER_COUNT}
-            max={MAX_PASSENGER_COUNT}
-            value={passengerCount}
-            onChange={changePassengerCount}
-          />
-          <button
-            className="passenger-count-control-button"
-            type="button"
-            aria-disabled={passengerCount === MAX_PASSENGER_COUNT}
-            aria-label="성인 승객 한명 늘이기"
-            onClick={increasePassengerCount}
-          >
-            +
-          </button>
-        </div>
-        <div className="user-action-message-box" role="status">
-          {message}
-        </div>
-      </section>
-    </main>
+    <S.Wrapper>
+      <h2>지금 떠나기 좋은 여행</h2>
+      <S.Container>
+        <Carousel>
+          {TravelInformation.map(({ route, href, id, image, price, seat }) => (
+            <li key={id}>
+              <a href={href}>
+                <S.TravelItem>
+                  <S.TravelItemDescription>
+                    <S.TravelItemRoute>{route}</S.TravelItemRoute>
+                    <S.TravelItemSeat>{seat}</S.TravelItemSeat>
+                    <S.TravelItemPrice>
```

Comment:
r: 스크린리더에서 KRW라고 읽어주네요. aria-label로 직접 레이블을 작성하면 좋을 것 같습니다!

```suggestion
                    <S.TravelItemPrice aria-label={`${price} 대한민국원`}>
```

Reply:
오우 감사합니다 크롬 데스크탑에서는 잘 작동했는데 모바일에서는 안되는군요... 수정했습니다!